proto/lib/config/deviceconfiguration.proto

```diff
@@ -24,4 +24,5 @@ message DeviceConfiguration {
     repeated ObservedFolder ignored_folders            = 14;
     repeated ObservedFolder pending_folders            = 15;
     int32                   max_request_kib            = 16 [(ext.goname) = "MaxRequestKiB", (ext.xml) = "maxRequestKiB", (ext.json) = "maxRequestKiB"];
+    int32                   remote_gui_port            = 17 [(ext.goname) = "RemoteGUIPort", (ext.xml) = "remoteGUIPort", (ext.json) = "remoteGUIPort"];
 }
```

Comment:
TCP ports are defined as uint16, so that's what I would specify when a type with explicit width is used.

Reply:
I forgot. Sorry

Reply:
I am not sure exactly what you mean. I thought protobuf had an uint16 type, but it seems to me it does not exist. https://developers.google.com/protocol-buffers/docs/proto3#scalar

Reply:
Apparently protobuf has deliberately left out any smaller fixed-width integer types.  So the application code would need to make sure the port numbers are within a sensible range.  On the other hand it's not really important as the browser will just fail on the probing if a port number larger than 65535 is configured.